.trybe/requirements.json

```
@@ -0,0 +1,60 @@
+{
+  "requirements": [
+    {
+      "description": "Sua lista deve possuir o título “Minha Lista de Tarefas” em uma tag <header>",
```

```suggestion
      "description": "Sua lista deve possuir o título \“Minha Lista de Tarefas\” em uma tag <header>",
```

Acho que tem q escapar essas aspas duplas aqui